packages/angular_devkit/build_angular/src/webpack/plugins/karma/karma.ts

```diff
@@ -124,10 +124,10 @@ const init: any = (config: any, emitter: any) => {
 
       logger.error(statsErrorsToString(statsJson, { colors: true }));
 
-      // Notify potential listeners of the compile error.
-      emitter.emit('compile_error', {
-        errors: statsJson.errors?.map((e) => e.message),
-      });
+      if (config.singleRun) {
+        // Notify potential listeners of the compile error.
+        emitter.emit('load_error');
```

Comment:
Why the change to the event name here?

Reply:
load_error is needed as otherwise in some cases karma will jot exit when there is a webpack error as karma doesn’t listen to compilation_error.